src/lib/createFaker.ts

```diff
@@ -51,11 +107,13 @@ export class Faker {
 	}
 
 	words(length: number, wordOffset = this.range(0, loremWords.length)): string {
-		return lorem(`${length}w`, wordOffset);
+		return length === 1 ? this.word() : lorem(`${length}w`, wordOffset);
```

Comment:
had to do this too because sometimes we use `words` with a random length, and if it's 1, then we have the same problem

i saw this exactly in the scenario we're trying to fix - for the id of the custom types we use a list of words of length 1-2 for the label and then snake case that label